doc/nrf/ug_bootloader_adding.rst

```diff
@@ -298,13 +298,13 @@ MCUboot is added to its boot chain by including the ``CONFIG_BOOTLOADER_MCUBOOT`
 |how_to_configure|
 
 MCUboot can use the cryptographic functionality exposed by the immutable bootloader, reducing the flash memory usage for MCUboot to less than 16 kB.
-To enable this configuration, apply the :file:`overlay-minimal-external-crypto.conf` Kconfig fragment file for the MCUboot image:
+To enable this configuration, apply the :file:`prj_minimal.conf` Kconfig fragment file for the MCUboot image:
```

Comment:
Since this fragment is a project file (`prj_*`), we should mention it as such instead of "Kconfig fragment file". Not sure on the proper wording between the two here, but the typical files we can Kconfig fragments, like what `overlay-minimal-external-crypto` was, are a different use case than project Kconfig files.